tee-worker/scripts/litentry/release/ReadMe.md

```diff
@@ -0,0 +1,97 @@
+
+# Release package
+
+
+## Step 1: Preparation
+
+From the root folder ~/litentry-parachain/tee-worker/:
+```
+make release-pkg
+```
+A release package will be generated, within which there are:
+
+- enclave.sign.so
+- integritee-service
+- running-mode-eg.json
+
+
+
+## Step 2: Deploy on production
+
+1. Extract the release package to one target location. Worker will be executed from there. Then execute `prepare.sh`:
```

Comment:
`... to one target location` => We need a concrete example.

You execute `prepare.sh` with `./`, which implies you are under `tee-worker/scripts/litentry/release/`. What's the relationship of the "target location"?

Reply:
Not really. This readme is prepared for release package.